lib/wasi/src/runtime/task_manager/tokio.rs

```diff
@@ -128,11 +128,27 @@ impl VirtualTaskManager for TokioTaskManager {
         // If we have a trigger then we first need to run
         // the poller to completion
         if let Some(trigger) = task.trigger {
+            tracing::trace!(
+                "wasm task will wait for trigger (pid={}, tid={})",
+                ctx.data(&store).thread.pid(),
+                ctx.data(&store).thread.tid()
```

Comment:
Let's use structured logging here - it'll make filtering with automated tools a lot easier later on because they get translated to field on a JSON object.

```suggestion
                pid=ctx.data(&store).thread.pid(),
                tid=ctx.data(&store).thread.tid(),
                 "wasm task will wait for trigger",
```